fili-core/src/main/java/com/yahoo/bard/webservice/web/apirequest/ApiRequest.java

```diff
@@ -73,43 +52,10 @@ default DateTimeFormatter generateDateTimeFormatter(DateTimeZone timeZone) {
      */
      long getAsyncAfter();
 
-    /**
-     * Get the response builder associated with this request.
-     *
-     * @return The response builder.
-     */
-     Response.ResponseBuilder getBuilder();
-
     /**
      * Get the default pagination parameters for this type of API request.
      *
      * @return The uri info of this type of API request
      */
      PaginationParameters getDefaultPagination();
-
-
-    /**
-     * Add links to the response builder and return a stream with the requested page from the raw data.
-     *
-     * @param <T>  The type of the collection elements
-     * @param data  The data to be paginated.
-     *
-     * @return A stream corresponding to the requested page.
-     *
-     * @deprecated Pagination is moving to a Stream and pushing creation of the page to a more general
-     * method ({@link #getPage(Pagination)}) to allow for more flexibility
-     * in how pagination is done.
-     */
-    @Deprecated
-     <T> Stream<T> getPage(Collection<T> data);
-
-    /**
-     * Add links to the response builder and return a stream with the requested page from the raw data.
-     *
-     * @param <T>  The type of the collection elements
-     * @param pagination  The pagination object
-     *
-     * @return A stream corresponding to the requested page.
-     */
-     <T> Stream<T> getPage(Pagination<T> pagination);
```

Comment:
Really nice 👍 I would like to remove them as well since I first saw them in this class.